frontend/src/App.tsx

```diff
@@ -5,18 +5,28 @@ import {BrowserRouter, Route, Routes} from "react-router-dom";
 import SpotDetail from "./components/SpotDetail";
 import useSpots from "./hooks/useSpots";
 import SpotList from "./components/spot_list/SpotList";
+import BottomNavigationBar from "./components/BottomNavigationBar";
+import {Spot} from "./models/Spot";
 
 function App() {
 
     const {mapboxToken} = useAccessToken()
-    const {spots} = useSpots()
+    const {spots, addSpot} = useSpots()
+
+    function handleAddSpot(newSpot:Spot):Promise<void> {
```

Comment:
`Promise<void>` Did you copy that from Fangfang? (Because we talke about this just this morning)
Keep it up! :wink:

Reply:
Nope came up with that myself 💪